core/ulog/doc.go

```diff
@@ -20,32 +20,37 @@
 
 // Package ulog is the Logging package.
```

Comment:
This language is coming from gendoc, by compiling "Logging package" into Package ulog is the Logging package. We will need to change the tool to recompile this line into more friendlier language.